try-catch-eip/public/manifest.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
Probably we should remove this file and both icons as well :)

Reply:
Done.